### PR TITLE
Output generated CSS at end of bundling

### PR DESCRIPTION
**Pull request by jesstelford**

### CODE REVIEW

File: plugin.js [hunk 1]
```diff
@@ -130,6 +133,18 @@ function cmifyPlugin (b, opts) {
         next(null, row)
       }
     }, function end (done) {
+
+      const outFile = opts.o || opts.outfile
+
+      if (outFile) {
+        try {
+          mkdirp.sync(path.dirname(outFile))
+          fs.writeFileSync(outFile, cmify.getAllCss())
```
Comment (jesstelford) [inline plugin.js:142]: Future enhancement; Detect if we're in `--verbose` mode, and if so; `console.log('Wrote to file ' + outFile)` or somesuch.

**Comment by jesstelford** [comment]
~~Note that when using this new option, the CSS is still inlined into `.getAllCss()`, so we're essentially doubling up. However, the user can tree shake this away using something like [rollupify](https://github.com/nolanlawson/rollupify)~~

Actually, after looking at the generated bundle, the plugin doesn't remove any `require('cmify')`s even if the only usage was `cmify.load()` which has since been replaced with `require('style.css')` etc. That's a good future enhancement; remove the `require('cmify')` if it's not used anymore in the file.

**Comment by joshwnj** [comment]
Looks good @jesstelford , just need to add `mkdirp` to package deps and then we're good to go :)

I like your suggestions for future enhancements too.

Thanks!

**Comment by jesstelford** [comment]
Whoops, missed that one. Have added and re-pushed.

How did you want to track the future enhancement? GitHub Issues?

**Comment by joshwnj** [comment]
@jesstelford cool thanks! I've added #6 and #7 to track those enhancements

**Comment by joshwnj** [comment]
Published v1.4.0

**Pull request merged**